ignite/cmd/testnet.go

```diff
@@ -7,12 +7,9 @@ import (
 // NewTestnet returns a command that groups scaffolding related sub commands.
 func NewTestnet() *cobra.Command {
 	c := &cobra.Command{
-		Use:   "testnet [command]",
-		Short: "Start a testnet local",
-		Long: `Start a testnet local
```

Comment:
It would be nice to have a long description of how to use the command. I will create the issue

Reply:
👍🏾